zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/ErrorMapperTest.java

```diff
@@ -51,6 +53,7 @@
       UserTaskController.class
     },
     webEnvironment = WebEnvironment.RANDOM_PORT)
+@AutoConfigureMockMvc(addFilters = false)
```

Comment:
❓ What is this needed for? Would the test fail without this?

Reply:
Really good point ^^ ... I don't like that either 😄 ... it's removing all filters that are used in the Spring Application. The aim was to not have Spring Security auto-configure and putting itself in place ... thus causing access problems when the test requests are made here...

I'm changing this right now, putting the property to disable the spring security auto configuration in this test instead. That's more specific and should do the job too. Will push in bit after sorting something else out which might have an influence on this too 😉